packages/at_secondary_server/pubspec.yaml

```diff
@@ -34,6 +34,13 @@ dependencies:
   yaml: 3.1.2
   logging: 1.2.0
 
+dependency_overrides:
```

Comment:
can you raise a separate PR for persistence

Reply:
since most of the changes in this PR are in at_persistence, I have removed one change that belonged to at_secondary, now this PR only contains changes for at_persistence.